src/core/ui.js

```diff
@@ -47,13 +47,15 @@ function ariaDecorate(elem, ariaMap) {
 
 const respecUI = hyperHTML`<div id='respec-ui' class='removeOnSave' hidden></div>`;
 const menu = hyperHTML`<ul id=respec-menu role=menu aria-labelledby='respec-pill' hidden></ul>`;
+const closeButton = hyperHTML`<span id="close-button">❌</span>`;
```

Comment:
```suggestion
const closeButton = hyperHTML`<span class="close-button">❌</span>`;
```

`class` is preferred since we don't use the id anywhere else. Also I would prefer putting this line at line 51 as currently line 49 and 50 are related and should be together.

Reply:
Can we use `button` here rather than `span`, so that screen readers can see whether this is a button without `aria-role`?